apps/dot-voting/test/dot-voting.test.js

```diff
@@ -634,6 +786,8 @@ contract('DotVoting App', accounts => {
         assert.equal(canExecute, false, 'canExecute should be false')
       })
       it('holder can add candidates', async () => {
+        let newvote = await app.newVote(script, 'metadata', { from: holder50 })
+        voteId = createdVoteId(newvote)
         mango = accounts[5]
         await app.addCandidate(voteId, '0xbeefdead', mango, 0x1, 0x1)
         candidates.push(mango)
```

Comment:
I don't think this really needs to be pushed/popped here. We can just use the local object.